pages/common/jello.md

```diff
@@ -0,0 +1,32 @@
+# jello
+
+> A command-line JSON processor using Python syntax.
+> More information: <https://github.com/kellyjonbrazil/jello>.
+
+- Pretty-print JSON or JSON-Lines data from stdin to stdout:
+
+`cat {{file.json}} | jello`
+
+- Output a schema of JSON or JSON Lines data from stdin to stdout (useful for grep):
+
+`cat {{file.json}} | jello -s`
+
+- Output all elements from arrays (or all the values from objects) in JSON or JSON-Lines data from stdin to stdout:
+
+`cat {{file.json}} | jello -l`
+
+- Output the first element in JSON or JSON-Lines data from stdin to stdout:
+
+`cat {{file.json}} | jello _[0]`
+
+- Output the value of a given key of each element in JSON or JSON-Lines data from stdin to stdout:
+
+`cat {{file.json}} | jello '[i.{{key_name}} for i in _]'`
```

Comment:
Ah, you can do Python list comprehension. Cool! 🚀

Reply:
Yep, basically anything you can do in Python. `jello` basically removes the boilerplate code you would need to ingest and output the filtered JSON in different formats. It's very similar to `jq`, which has its own quirky syntax, so I wrote `jello` to work very much like `jq` but with Python syntax.